### ISSUE TITLE
Add support for numa_interleave

### ISSUE THREAD

**Issue by tersmitten** [Oefenweb/ansible-percona-server#11]
https://www.percona.com/doc/percona-server/5.5/performance/innodb_numa_support.html#numa_interleave

**Comment by mvdriel** [comment]
In #14 it is possible to create more flexible MySQL configuration. For example:

```
percona_server_my_cnf:
  - section: mysqld
    options:
      - name: numa_interleave
        value: 1
```

Does this fix this issue?

**Comment by tersmitten** [comment]
Fixed in #17
